models/article.models.js

```diff
@@ -36,3 +36,29 @@ exports.selectArticleById = (article_id) => {
       throw err;
     });
 };
+
+exports.selectCommentsByArticleId = (article_id) => {
+  return db
+    .query(
+      "SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC;",
+      [article_id]
+    )
+    .then((result) => {
+      if (result.rows.length === 0) {
+        return Promise.reject({
+          status: 404,
+          msg: `No comments found for article_id: ${article_id}`,
+        });
+      }
```

Comment:
As I mentioned within the test file the comments coming back as empty would not be an error.
But please add a test to show that you do infact get an empty array returned.